### PR TITLE
Added AlphaVantage

### PR DESCRIPTION
**Pull request by OhadVal**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
New pack - AlphaVantage for fetching stock data.
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
+"""
+AlphaVantage is a stock data API TODO: MA?
```
Comment (Itay4) [inline Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.py:2]:
```suggestion
AlphaVantage is a stock data API
```

File: Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
+category: Utilities
+commonfields:
+  id: AlphaVantage
+  version: -1
+configuration:
+- display: API Key
+  name: api_key
+  required: true
+  type: 4
+- display: Fetch incidents
```
Comment (Itay4) [inline Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.yml:10]: The fetch related params can be removed

File: Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+            error_handler=error_handler, empty_valid_codes=empty_valid_codes, **kwargs
+        )
+        if error_message := response.get('Error Message'):
+            raise DemistoException(error_message)
+        return response
+
+    def get_stock_history(
+            self, symbol: str, interval: str, output_size: str = 'compact'
+    ) -> dict:
+        """TODO
```
Comment (Itay4) [inline Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.py:62]: Do you wish to handle these TODOs ?

File: Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.yml [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+
+
+fromversion: 5.0.0
```
Comment (Itay4) [inline Packs/AlphaVantage/Integrations/AlphaVantage/AlphaVantage.yml:139]:
```suggestion
fromversion: 6.0.0
```

File: Packs/AlphaVantage/Integrations/AlphaVantage/test_data/stock history get.json [hunk 5]
```diff
@@ -0,0 +1,712 @@
+{
```
Comment (Itay4) [inline Packs/AlphaVantage/Integrations/AlphaVantage/test_data/stock history get.json:1]: The test data files are not used anywhere, did you plan to add unit tests for these or can be removed?

File: Packs/AlphaVantage/Playbooks/Notify_Stock_Above_Price.yml [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+    complex:
+      root: incident
+      accessor: username
+  required: true
+  description: The username of the telegram account to send a message to.
+  playbookInputQuery:
+outputs: []
+tests:
+- AlphaVantage Test Playbook
+fromversion: 5.0.0
```
Comment (Itay4) [inline Packs/AlphaVantage/Playbooks/Notify_Stock_Above_Price.yml:304]:
```suggestion
fromversion: 6.0.0
```

**Review by Itay4 — COMMENTED** [review]
Thank you for your contribution !
Overall looks good, added some minor comments

File: Packs/AlphaVantage/Integrations/AlphaVantage/test_data/stock history get.json (hunk 5 above)
Comment (OhadVal) [reply Packs/AlphaVantage/Integrations/AlphaVantage/test_data/stock history get.json:1]: I did not plan to add more unit tests, those files are nice to have example for the API responses.

File: Packs/AlphaVantage/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "AlphaVantage",
+    "description": "API for stocks prices data",
```
Comment (sangsoojun88) [inline Packs/AlphaVantage/pack_metadata.json:3]: -> The Alpha Vantage content pack provides accessible APIs for financial market data such as stock prices. Utilize this pack to get open stock prices, high/low price, trade volume, and so on.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**